hypertrace-core-graphql-log-event-schema/src/main/java/org/hypertrace/core/graphql/log/event/request/DefaultLogEventRequestBuilder.java

```diff
@@ -1,22 +1,152 @@
 package org.hypertrace.core.graphql.log.event.request;
 
+import static io.reactivex.rxjava3.core.Single.zip;
+
+import graphql.schema.DataFetchingFieldSelectionSet;
+import graphql.schema.SelectedField;
 import io.reactivex.rxjava3.core.Single;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
 import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import javax.inject.Inject;
 import lombok.Value;
 import lombok.experimental.Accessors;
+import org.hypertrace.core.graphql.common.request.AttributeAssociation;
+import org.hypertrace.core.graphql.common.request.AttributeRequest;
+import org.hypertrace.core.graphql.common.request.AttributeRequestBuilder;
+import org.hypertrace.core.graphql.common.request.FilterRequestBuilder;
+import org.hypertrace.core.graphql.common.schema.arguments.TimeRangeArgument;
+import org.hypertrace.core.graphql.common.schema.results.ResultSet;
+import org.hypertrace.core.graphql.common.schema.results.arguments.filter.FilterArgument;
+import org.hypertrace.core.graphql.common.schema.results.arguments.order.OrderArgument;
+import org.hypertrace.core.graphql.common.schema.results.arguments.page.LimitArgument;
+import org.hypertrace.core.graphql.common.schema.results.arguments.page.OffsetArgument;
+import org.hypertrace.core.graphql.common.utils.attributes.AttributeAssociator;
 import org.hypertrace.core.graphql.context.GraphQlRequestContext;
+import org.hypertrace.core.graphql.deserialization.ArgumentDeserializer;
+import org.hypertrace.core.graphql.utils.schema.GraphQlSelectionFinder;
+import org.hypertrace.core.graphql.utils.schema.SelectionQuery;
 
 public class DefaultLogEventRequestBuilder implements LogEventRequestBuilder {
 
+  private final ArgumentDeserializer argumentDeserializer;
+  private final GraphQlSelectionFinder selectionFinder;
+  private final AttributeRequestBuilder attributeRequestBuilder;
+  private final AttributeAssociator attributeAssociator;
+  private final FilterRequestBuilder filterRequestBuilder;
+  private static final int DEFAULT_LIMIT = 100;
+  private static final int DEFAULT_OFFSET = 0;
+
+  @Inject
+  DefaultLogEventRequestBuilder(
```

Comment:
This is still identical to the result set request builder right? Forgot where that conversation landed, can we just reuse the existing builder and request?

Reply:
id attribute is the difference, so in DefaultResultSetRequestBuilder it tries to call org.hypertrace.core.graphql.common.request.AttributeRequestBuilder#buildForAttributeQueryableFieldsAndId

Reply:
ah right, gotcha.